tests/test_config.json

```diff
@@ -0,0 +1,42 @@
+{ "train_data_path": "/home/neele/PycharmProjects/ambigous-alpaca/tests/data_multiclassification/train.txt",
```

Comment:
hardcoded paths

Reply:
yes is the configuration file which is basically the only file where we want to hardcode stuff but we should take the relative path here because that should be the same for both of us...

Reply:
that's what I meant... (my fifu embeddings are back!)